Sources/App/routes.swift

```diff
@@ -259,16 +259,20 @@ func routes(_ app: Application) throws {
     }
 
     do {  // RSS + Sitemap
-        app.get(SiteURL.rssPackages.pathComponents, use: RSSFeed.showPackages)
-            .excludeFromOpenAPI()
-
-        app.get(SiteURL.rssReleases.pathComponents, use: RSSFeed.showReleases)
-            .excludeFromOpenAPI()
+        app.group(BackendReportingMiddleware(path: .rss)) {
+            $0.get(SiteURL.rssPackages.pathComponents, use: RSSFeed.showPackages)
+                .excludeFromOpenAPI()
+            
+            $0.get(SiteURL.rssReleases.pathComponents, use: RSSFeed.showReleases)
+                .excludeFromOpenAPI()
+        }
```

Comment:
I've collapsed both rss feed paths (the actual ones are `/packages.rss` and `releases.rss` into `/rss` for sake of simplicity. We could also report them individually under some path, perhaps with a `/rss/` prefix to make filtering easier 🤷‍♂️

Reply:
I think this is fine as one trigger. No benefit in splitting these out. 👍